lstchain/visualization/plot_drs4.py

```diff
@@ -42,24 +42,31 @@ def plot_pedestals(data_file, pedestal_file, run=0 , plot_file="none", tel_id=1,
 
     plt.rc('font', size=15)
```

Comment:
I opened a PR some time ago cause I think the `PdfPages` object is created twice in this script, here and the same lines appear later on around L90. I think the PDF should be opened only once.